### ISSUE TITLE
Unable to sign in on staging

### ISSUE THREAD

**Issue by thienlnam** [Expensify/App#10917]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to https://staging.new.expensify.com/
2. Logout if already logged in
3. Enter your email click on continue button

## Expected Result:
User should be able to log in

## Actual Result:
Describe what actually happened
Infinite spinner
<img width="1505" alt="Screen Shot 2022-09-09 at 10 14 41 AM" src="https://user-images.githubusercontent.com/30609178/189315904-a293bc63-b237-4685-86bc-f0378a2d826e.png">

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?
No

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 
**Reproducible in staging?:** 
**Reproducible in production?:** 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:** 
**Issue reported by:** @mdneyazahmad
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1662709037179189

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by thienlnam** [comment]
Looks like the errors from this CSP rule here: https://github.com/Expensify/Cloudflare-Workers/blob/26d94b8fc324d03c8817b6860726c0ebc41770be/new.expensify.com/wrangler.toml

**Comment by thienlnam** [comment]
Related issue: https://github.com/Expensify/App/issues/10163
PR: https://github.com/Expensify/App/pull/10690

cc @mountiny 
Looks like we are running into CSP errors with these changes

**Comment by mountiny** [comment]
Thanks for the ping, I will create a revert PR. Not sure if we can fix this fast

**Comment by thienlnam** [comment]
I think we just need to try to update the CSP rule and can fix this

**Comment by mountiny** [comment]
> CSP rule and can fix this

Ah, where is that defined 🤔

**Comment by thienlnam** [comment]
Yeah, let's revert these changes and then make a PR to update CSP rules and then we can re-do these changes

**Comment by mountiny** [comment]
Tested logging in staging after the revert and worked good.

**Comment by mountiny** [comment]
@kevinksullivan Would you be able to pay out @mdneyazahmad for reporting this bug. Thank you!

**Comment by dylanexpensify** [comment]
I can tackle this!

**Comment by dylanexpensify** [comment]
internal: https://www.upwork.com/ab/applicants/1569301205602381824/job-details
external: https://www.upwork.com/jobs/~01f0713bffe19757cae

**Comment by dylanexpensify** [comment]
@mdneyazahmad ^ 😄

**Comment by dylanexpensify** [comment]
https://www.upwork.com/jobs/~01f0713bffe19757ca huh - try this link? Or send me your name and I'll invite you to the job!

**Comment by dylanexpensify** [comment]
Offer sent!
